packages/RichTextEditing-Core.package/TextStructure.class/instance/initialize.st

```diff
@@ -2,7 +2,6 @@ initialize-release
 initialize
 
 	super initialize.
-	self 
-		initializeStyles;
-		color: nil;
-		font: nil
\ No newline at end of file
+	self
+		attributes: Set new;
+		preAddHook: [:text | ]
```

Comment:
Why do we have to add an empty hook here? I think this should be checked elsewhere

Reply:
The alternative would be to do lazy initialization in the getter: `^ preAddHook ifNil: [[:text | ]]`. Isn't initialize the place to initialize values, especially since we don't do lazy initialization elsewhere?